src/Configuration.test.ts

```diff
@@ -81,3 +81,20 @@ describe('src/Configuration', async (should: any) => {
     }
   }
 })
+
+describe('loadConfigurationWithDefaults', async (should: any) => {
```

Comment:
Since `loadConfigurationWithDefaults`, isn't this more an integration test than a unit test?